crates/prql_compiler/src/lib.rs

```diff
@@ -305,12 +307,12 @@ pub mod json {
     use super::*;
 
     /// JSON serialization
-    pub fn from_pl(pl: Vec<ir::pl::Stmt>) -> Result<String, ErrorMessages> {
+    pub fn from_pl(pl: Vec<prql_ast::stmt::Stmt>) -> Result<String, ErrorMessages> {
         serde_json::to_string(&pl).map_err(|e| anyhow::anyhow!(e).into())
     }
 
     /// JSON deserialization
-    pub fn to_pl(json: &str) -> Result<Vec<ir::pl::Stmt>, ErrorMessages> {
+    pub fn to_pl(json: &str) -> Result<Vec<prql_ast::stmt::Stmt>, ErrorMessages> {
         serde_json::from_str(json).map_err(|e| anyhow::anyhow!(e).into())
     }
```

Comment:
I think we should actually get rid of these functions so that we can drop the `serde_json` dependency.

Reply:
IIRC, bindings use these. As does the playground...